### DISCUSSION TITLE
First github experience?

### DISCUSSION THREAD

**Discussion by ezmarche** [peppedilillo/mescal#3] [Ideas]
Most of us (myself included) are new to github. So I propose we use this Discussion to suggest how to exploit the github capabilities at its best :)

Ez.-

**Comment by peppedilillo** [comment]
hello eze, ty for having set everything up, it's super cool!!!! one thing which i feel is important: i'd avoid using different branches for different use cases. branches are for experimenting with code without doing damage.  i'd suggest to remove the branches Smode, Specutilities and Xmode. after that we'll think of some scripts or simple interface to deal with the different sw use cases. 

ciao,
p
